docgen/assets/js/examples/ecommerce.js

```diff
@@ -184,7 +185,14 @@ function CustomHits(props) {
   );
 }
 
-function CustomResults(props) {
+const CustomResults = createConnector({
+  displayName: 'CustomResults',
+
+  getProps(props, state, search) {
```

Comment:
Nice in the end even if the naming/way to do this is not obvious right now, you still managed to do something very small